subsys/fs/nvs/nvs.c

```diff
@@ -555,7 +554,7 @@ static int nvs_sector_close(struct nvs_fs *fs)
 
 	nvs_ate_crc8_update(&close_ate);
 
-	rc = nvs_flash_ate_wrt(fs, &close_ate);
+	(void)nvs_flash_ate_wrt(fs, &close_ate);
```

Comment:
The return value of `nvs_sector_close` is actually checked at `nvs_write`, where code skips to end when this is not 0.
If this gets discarded the code in `ncs_write` no longer makes sense.
Shouldn't this be rather properly returned by `ncs_sector_close`?

Reply:
It could return the error code, but this change does not modify the present behavior.

nvs is (should be) resistant to unfinished sector_close writes as it cannot be guaranteed that a close is properly written during a powerdown.

Reply:
Still there is code that uses the return value, so we either need to do something here or where this is called.

Reply:
Why ? The return value was always 0 anyhow. A further simplication could be done (this was always the case) but if there ever is a real reason to use the return value it will be much more complex to revert this. This code has been constant for quite some time and I do not see a real reason to change it.

Reply:
As far as I can tell the NVS is not supported anymore so it is on a death path.